python/meterpreter/ext_server_stdapi.py

```diff
@@ -879,6 +879,14 @@ def ctstruct_unpack(structure, raw_data):
     ctypes.memmove(ctypes.byref(structure), raw_data, ctypes.sizeof(structure))
     return structure
 
+def get_process_output(args):
+    proc_h = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = proc_h.communicate()
```

Comment:
Method manually tested with 2.4, as I couldn't get pip/mock installed with an older version of python

```python
# cat test.py 
import subprocess

def get_process_output(args):
    proc_h = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
    stdout, stderr = proc_h.communicate()

    if proc_h.wait():
        raise Exception(args[0] + ' exited with non-zero status')
    return str(stdout)


print(get_process_output(['ps', 'ax', '-w', '-o', 'pid,ppid,user,command']))
```

```
# python2.4 test.py 
  PID  PPID USER     COMMAND
    1     0 root     /bin/bash
 2157     1 root     python2.4 test.py
 2158  2157 root     ps ax -w -o pid,ppid,user,command
```